pkgs/by-name/dc/dclock/package.nix

```
@@ -35,11 +35,11 @@ haskellPackages.mkDerivation {
     process
   ];
 
-  meta = {
+  meta = with lib; {
```

```suggestion
  meta = {
```
It's mostly advised you use `with lib;` only in places it is required like license and maintainers.